spec/controllers/line_items_controller_spec.rb

```diff
@@ -32,9 +32,13 @@
       order = create(:completed_order_with_totals)
       item = create(:line_item, order: order)
       while !order.completed? do break unless order.next! end
+      order.reload
```

Comment:
I've made two commits. You should just be able to cherry-pick the second one into your upgrade branch.